libs/game/systemmenu.ts

```diff
@@ -269,16 +269,18 @@ namespace scene.systemMenu {
 
     function volumeDown() {
         const v = music.volume();
-        music.setVolume(v - 32);
+        const newVolume = ((v - 1) >> 5) << 5;
```

Comment:
when v = 0, newVolume is -32. is that expected?

Reply:
Can we do less magic for this?

Reply:
Yeah, I was letting setVolume ignore it. I'll rewrite it to be less bit shifty